tvtk/vtk_parser.py

```diff
@@ -625,16 +625,20 @@ def _find_get_set_methods(self, klass, methods):
                 (klass_name == 'vtkImageDataGeometryFilter' and
                  method[3:] == 'Extent') or
                 (klass_name == 'vtkVolumeMapper' and
-                 method[3:] == 'CroppingRegionPlanes') or
-                (klass_name == 'vtkContextMouseEvent' and
-                 method[3:] == 'Interactor')):
+                 method[3:] == 'CroppingRegionPlanes')):
+                continue
+            elif (klass_name == 'vtkContextMouseEvent' and
+                  method[3:] == 'Interactor'):
                 continue
             # VTK 9 uses function handles that we don't parse properly yet
             elif (klass_name == 'vtkPiecewisePointHandleItem' and
                   method[3:] == 'PiecewiseFunction'):
                 continue
             # we can actually process it
             elif ('Get' + method[3:]) in methods:
+                # import sys
```

Comment:
Could you please remove these commented lines?

Reply:
Sure, but it's useful as a companion to these very similar lines when debugging is necessary to figure out where segfaults are coming from:

https://github.com/enthought/mayavi/blob/master/tvtk/tests/test_vtk_parser.py#L307-L308